nifi-nar-bundles/nifi-standard-services/nifi-hbase_1_1_2-client-service-bundle/nifi-hbase_1_1_2-client-service/src/main/java/org/apache/nifi/hbase/HBase_1_1_2_ClientService.java

```diff
@@ -88,16 +88,16 @@
 
     static final long TICKET_RENEWAL_PERIOD = 60000;
 
-    private volatile Connection connection;
+    protected volatile Connection connection;
     private volatile UserGroupInformation ugi;
     private volatile KerberosTicketRenewer renewer;
 
-    private List<PropertyDescriptor> properties;
-    private KerberosProperties kerberosProperties;
+    protected List<PropertyDescriptor> properties;
```

Comment:
Can we leave this as private and then add a protected method that sub-classes can use to add additional properties? So something like this...
```
protected List<PropertyDescriptor> getAdditionalProperties() {
}
```
And then in init, it would call it like this:
```
List<PropertyDescriptor> props = new ArrayList<>();
        ...
        props.addAll(getAdditionalProperties());
        this.properties = Collections.unmodifiableList(props);
```

Then the lookup service would override getAdditionalProperties() to provide it's specific properties to be added.